modules/remotebackend/remotebackend.cc

```diff
@@ -922,6 +922,28 @@ bool RemoteBackend::searchComments(const string &pattern, int maxResults, vector
   return false;
 }
 
+bool RemoteBackend::getSubZones(const string &pattern, vector<std::tuple<string, string>>& result)
+{
+    Json query = Json::object{
+    { "method", "getSubZones" },
+    { "parameters", Json::object{
+      { "pattern", pattern }
```

Comment:
`pattern` is a parent zone name here, right?
if so, a different name would be a lot better.